DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/lacp/test_lacp_routing_over_bridge.py

```diff
@@ -147,4 +147,4 @@ async def test_lacp_routing_over_bridge(testbed):
     stats = await tgen_utils_get_traffic_stats(tgen_dev, 'Traffic Item Statistics')
     for row in stats.Rows:
         err_msg = f"Expected 0.00 loss, actual {float(row['Loss %'])}"
-        assert tgen_utils_get_loss(row) == 0.000, err_msg
+        assert tgen_utils_get_loss(row) < 0.1, err_msg
```

Comment:
tolerances are good but sometimes dangerous
if rate is correct and all
why would any packet get lost?

for example whe na device is freshly booted and FDB is empty if you send too much traffic some devices will drop it.
but that can be fixed in script by sending a "learning" portion to alow FDB to be populated and than send the test traffic and expecting o loss

or if traffic is flowing, there can be packets in flight and tx will not perfectly match rx, but stoppi

any way this < 0.1 is solving the intermittent failures now but may hide real bugs in future.